source/_posts/2019/2019-01-07-how-to-create-your-first-php-twig-static-website-under-2-minutes-with-statie.md

```diff
@@ -3,8 +3,8 @@ id: 174
 title: "How to Create Your First PHP Twig Static Website under 2 Minutes with Statie"
 perex: |
     Do you like to write or create micro-sites? This post is for you.
-    Statie is now [the most downloaded](https://packagist.org/packages/symplify/statie/stats) PHP Twig static site generators, even surpassing 7-years old [Sculpin](https://packagist.org/packages/sculpin/sculpin/stats) by 200 downloads a month. On the other hand, Sculpin is about to release [version 3](https://github.com/sculpin/sculpin/releases) creating healthy competence.  
-    <br><br>
```

Comment:
Ah, I see - I did that because in the generated code there were in fact three `<br>`s and the spacing was a bit too much. However, by looking at the previous post ther are two in the generated and in the source - weird :)

- https://www.tomasvotruba.cz/blog/2019/01/03/how-to-complete-type-declarations-without-docblocks-with-rector/
- https://github.com/TomasVotruba/tomasvotruba.cz/edit/master/source/_posts/2019/2019-01-03-how-to-complete-type-declarations-without-docblocks-with-rector.md

Reply:
Added the `<br>` back - it should be in the source, the output is wonky :)

Reply:
Oh, I see. I shall explore this later, it should be consistent with previous posts.

Thanks for pointing this out :)

Reply:
Here was the catch: https://github.com/TomasVotruba/tomasvotruba.cz/commit/f9b87ec59caa22cbdb8191d9f9395a71c639aeac#diff-df7ac57a25c85f4195dc79c5fc355da2L7

Extra space :)